scripts/oss-fuzz.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# Copyright 2017 Jonathan Rudenberg
+#
+# systemd is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# systemd is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with systemd; If not, see <http://www.gnu.org/licenses/>.
+
+set -ex
+
+export LC_CTYPE=C.UTF-8
+
+meson $WORK -Doss-fuzz=true -Db_lundef=false -Dseccomp=false -Dlz4=false -Dlibiptc=false -Dlibidn=false
+ninja -C $WORK fuzzers
+
+# get DNS packet corpus
+df=$WORK/dns-fuzzing
+rm -rf $df
+git clone --depth 1 https://github.com/CZ-NIC/dns-fuzzing $df
```

Comment:
Always cloning this from scratch seems a bit wasteful, but I don't know what the best here would be. So maybe let's leave that as is for now.

Reply:
I don't think oss-fuzz caches $WORK, so it doesn't matter.